test/Microsoft.AspNet.FileProviders.Physical.Tests/PhysicalFileProviderTests.cs

```diff
@@ -669,7 +680,7 @@ public void Tokens_With_Forward_And_Backward_Slash()
         }
 
         [ConditionalFact]
-        [FrameworkSkipCondition(RuntimeFrameworks.Mono)]
+        [OSSkipCondition(OperatingSystems.Linux, SkipReason = "Race condition in inotify API makes this very flaky.")]
```

Comment:
Was there no reasonable amount of delay that would make this reliable? How long would you need?